### DISCUSSION TITLE
How to delete all scripts with one click?

### DISCUSSION THREAD

**Discussion by 20171213** [violentmonkey/violentmonkey#1460] [Q&A] [answered]
Don't reset, just click to delete one by one?

**Comment by gera2ld** [comment]
Press <kbd>Down</kbd>, then keep pressing <kbd>x</kbd> until all scripts deleted.

BTW, why would you do that?

**Comment by tophf** [comment, marked as answer]
You can also reinstall the extension.

Another method is to run the following code in devtools console for Violentmonkey dashboard (options) page:
```js
chrome.storage.local.get(all =>
  chrome.storage.local.remove(Object.keys(all).filter(k => k.includes(':')), () =>
    chrome.runtime.reload()))
```
